components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -680,11 +679,11 @@ impl Interpolate for BoxShadow {
     }
 }
 
-impl Interpolate for LengthOrNone {
+impl<T: Copy> Interpolate for Either<Au, T> {
```

Comment:
Why the `T: Copy` bound? Seems quite arbitrary. I'd prefer a generic `Interpolate` for `Either<T, U>`, if that's possible, that may very well not be.